src/content/docs/en/chromebook/faq-and-troubleshooting.mdx

```diff
@@ -0,0 +1,47 @@
+---
+title: Troubleshooting and FAQs
+description: "Troubleshooting tips and tricks, and FAQs"
+---
+
+import Alert from "../../../../components/Docs/Alert.astro";
+
+## Troubleshooting
+
+#### Chromebook fails to boot to internal disk
+
+Assuming you are selecting 'Boot to Internal Disk', or pressing <kbd>ctrl</kbd>+<kbd>D</kbd>> does not fully boot, this usually means the device has died. Plug in your Chromebook. If the issue continues to persist, hop into [our chats](/en/community/community) and make a support forum post.
+
+#### Keyboard is not properly mapped
+
+Try running Weirdtreething's keyboard [remap script](https://github.com/WeirdTreeThing/cros-keyboard-map) (creates a Keyd config)
+
+#### Chromebook audio doesn't work
+
+Try running Weirdtreething's [audio script](https://github.com/WeirdTreeThing/chromebook-linux-audio) manually.
+
+You may have a speaker unsupported by the audio script (max98357a speakers). If you would like to enable audio on here, run the script manually and follow the prompts to enable audio.
+
+<Alert type="warning">
+  max98357a, and any other audio devices requiring ```--force-avs-install``` is
+  not recommended to enable, and support will not be given to damaged speakers.
+  If you really want to proceed, do not raise your speakers volume to high or
+  they will get damaged.
+</Alert>
+
+## FAQs
+
+#### Is this legal?
+
+Yes, it is perfectly legal. ChromeOS and Chromebooks are backed by the [hromeium Projects' ChromiumOS](https://www.chromium.org/chromium-os/}). All information on how to mod Chromebooks is on this site, and open to anyone to use.
+
+#### Why not just run [mrchromebox's Firmware Utility Script?](https://mrchromebox.tech/#fwscript)
+
+The idea here is to be as easy and safe to use as possible. It is also easier to go back to ChromeOS if you keep the stock firmware. With that being said, we fully support the use of mrchromebox's script, it has many great use cases (This FAQ page is even being written on a full UEFI ROM Chromebook).
+
+#### Will this make my Chromebook faster?
+
+It usually can, as Linux is more light weight, but we do not garuntee preformance gains vs ChromeOS. If you are using an older/slower Chromebook, we reccomend using Xfce or KDE edition.
```

Comment:
lightweight is a compound work, but say "lighter" instead

also its guarantee, i can't spell it either it's fine

also "improvements" instead of "gains" and suggest Flagship as it is lighter than KDE